### PR TITLE
Feature/create subnewick

### LINKED ISSUE #259: Retrieve subgraph from existing Newick tree
As a user I want to highlight paths in the graph of certain strands.

### PR DESCRIPTION
**Pull request by jboot**
Not really sure about the line/branch coverage, since mvn site doesn't seem to work.
Fixes #259

### CODE REVIEW

File: geex-core/geex-core.iml [hunk 1]
```diff
@@ -12,6 +12,7 @@
     </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="module" module-name="geex-services" />
```
Comment (Vennik) [inline geex-core/geex-core.iml]: Duplicate,

File: geex-core/src/main/java/nl/tudelft/context/controller/SelectNewickController.java [hunk 2]
```diff
@@ -42,28 +43,32 @@ public SelectNewickController(final GraphController graphController,
...
-                .collect(Collectors.toList());
-
-        // Bind nodes
-        List<Label> nodeList = newick.vertexSet().stream()
-                .map(NewickLabel::new)
-                .collect(Collectors.toList());
-
-        this.newick.getChildren().addAll(edgeList);
-        this.newick.getChildren().addAll(nodeList);
+        try {
```
Comment (Vennik) [inline geex-core/src/main/java/nl/tudelft/context/controller/SelectNewickController.java]: Why is this done with try catch?

File: geex-core/src/main/resources/application/css/style.css [hunk 3]
```diff
@@ -181,22 +181,26 @@
 }
 
 /* Newick */
-#newick {
+#newickContainer {
```
Comment (Vennik) [inline geex-core/src/main/resources/application/css/style.css:198]: css id's are done with a '-' in this project.
And is standard in most modern frameworks.

File: geex-services/geex-services.iml [hunk 4]
```diff
@@ -7,7 +7,6 @@
       <sourceFolder url="file://$MODULE_DIR$/src/main/java" isTestSource="false" />
       <sourceFolder url="file://$MODULE_DIR$/src/test/java" isTestSource="true" />
       <sourceFolder url="file://$MODULE_DIR$/src/test/resources" type="java-test-resource" />
-      <sourceFolder url="file://$MODULE_DIR$/src/main/resources" type="java-resource" />
```
Comment (Vennik) [inline geex-services/geex-services.iml:10]: I prefer to have those as placeholder in case you need them.

File: geex-workspace/geex-workspace.iml [hunk 5]
```diff
@@ -6,7 +6,6 @@
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/main/java" isTestSource="false" />
       <sourceFolder url="file://$MODULE_DIR$/src/test/java" isTestSource="true" />
-      <sourceFolder url="file://$MODULE_DIR$/src/main/resources" type="java-resource" />
```
Comment (Vennik) [inline geex-workspace/geex-workspace.iml:9]: Same as geex-services.iml

**Comment by Vennik** [comment]
That's a big one :). Looks good. Added some comments though.

**Comment by jboot** [comment]
I fixed the try/catch part; wasn't needed anymore. Thank you for spotting that one.
I, however:
- am not sure what is wrong with the `.iml` files that you commented on;
- find camelCase more convenient for id's, since JavaFX uses the id for matching declared `@FXML` fields. I also didn't find any counterexamples in `style.css`.

**Comment by Vennik** [comment]
:shipit:

**Comment by jboot** [comment]
Also closes #260 now I think of it

**Pull request merged**